e2e/playwright.config.ts

```diff
@@ -46,7 +46,7 @@ export default defineConfig({
   outputDir: './test-results',
   timeout: 60000,
   use: {
-    baseURL: process.env.BASE_URL,
+    baseURL: process.env.BASE_URL ?? 'http://localhost:8888',
```

Comment:
Part of the goal of setting the defaults in the `.env.example` is to not having to repeat it everywhere in the code..